packages/core/src/checks/firehose_is_array.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const _ = require('lodash');
+const { simpleTruncate } = require('../tools/data');
+
+const isFirehoseWebhook = require('./is-firehose-webhook');
+
+/*
+  The firehoseWebhook performSubscriptionKeyList function should always return an array of objects.
+*/
+const firehoseWebhookIsArray = {
```

Comment:
I tried testing this locally, but didn't get this to error. I also tried the trigger is array and it also didn't error (when I deliberately tried to make it). Not sure if there's a particular way to test this locally?

Reply:
You registered the check correctly, so i'm not sure. It should get run whenever your `perform` is called, so nothing special should be needed. Though, your local app needs to be using this version of core, so that could be it.

We've also got unit tests for these checks, so that should be able to verify the behavior.